### PR TITLE
Screen Class

### PR DESCRIPTION
**Pull request by MASNathan**
Object oriented implementation of screen

Packagist: https://packagist.org/packages/screen/screen

Heroku App: https://stormy-waters-5943.herokuapp.com/

### CODE REVIEW

**Comment by peter-mw** [comment]
Hi, very nice work

I will test on local and merge if it works.

Do you want me to add you as a project maintainer?
This project receives some attention and i don't have the time to add more features.

If someone is willing to take care of maintaining this project i would be very glad.

**Comment by MASNathan** [comment]
Hello Peter,

I would very much like so.

I'll be awaiting for the privilegie.
Em 11/01/2016 21:19, "Peter Ivanov" <notifications@github.com> escreveu:

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "screen/screen",
```
Comment (peter-mw) [inline composer.json]: pls change to microweber/screen to mach the repo name

**Comment by peter-mw** [comment]
hi @MASNathan i have added you as a project admin

**Comment by MASNathan** [comment]
Hello @peter-mw
Sorry for the delayed reply, I was on my honeymoon.

In the next couple of days I'll do some stuff

**Pull request merged**